doc/source/flexibility/binlogging_replication_improvements.rst

```diff
@@ -118,6 +118,158 @@ about what else affects the writing of ``FLUSH`` commands to the binary log.
       |MySQL| Documentation: FLUSH Syntax
          https://dev.mysql.com/doc/refman/8.0/en/flush.html
 
+.. _percona-server.binary-log.UDF:
+
+Binary Log User Defined Functions
+================================================================================
+
+To implement Point in Time recovery, we have added the ``binlog_utils_udf``. The following user-defined functions are included:
+
+.. list-table::
+   :widths: 20 20 50
+   :header-rows: 1
+
+   * - Name
+     - Returns
+     - Description
+   * - get_binlog_by_gtid()
+     - Binlog file name as STRING
+     - Returns the binlog file name that contains the specified GTID
+   * - get_last_gtid_from_binlog()
+     - GTID as STRING
+     - Returns the last GTID found in the specified binlog
+   * - get_gtid_set_by_binlog()
+     - GTID set as STRING
+     - Returns all GTIDs found in the specified binlog
+   * - get_binlog_by gtid_set()
+     - Binlog file name as STRING and GTIDs as STRING
+     - Returns the binlog file name which contains at least one GTID and all GTIDs which are stored inside the returned binlog file
+   * - get_first_record_timestamp_by_binlog()
+     - Timestamp
+     - Returns the first recorded timestamp in the specified binlog
+   * - get_last_record_timestamp_by_binlog()
+     - Timestamp
+     - Returns the last recorded timestamp in the specified binlog
+
```

Comment:
Here, after the summary table please note the following:
1. All functions returning timestamps return their values as microsecond precision Unix Time. In other words,  they represent the number of microseconds since 1-JAN-1970.
2. All functions accepting binlog name as parameter accept only short names (without path component). If the path separator '/' is found in the input, an error is returned. This serves the purpose of restricting locations from where binlogs can be read from. They are always read from the current binlog directory (@@log_bin_basename system variable).
3. All functions returning binlog file names return them in short form (without path component).

Reply:
Added.